cmd/zfs/zfs_main.c

```diff
@@ -6808,6 +6820,7 @@ typedef struct share_mount_state {
 	uint_t	sm_total; /* number of filesystems to process */
 	uint_t	sm_done; /* number of filesystems processed */
 	int	sm_status; /* -1 if any of the share/mount operations failed */
+	boolean_t	sm_explicit;
```

Comment:
Does this struct need re-aligning, and does this field need a comment? Also, should this be called `sm_recursive` instead of `sm_explicit`?

Reply:
Thanks!